### ISSUE TITLE
Update notebooks file to work with latest master

### ISSUE THREAD

**Issue by T0T4R4** [timothydmorton/isochrones#111]
I can't find a single one of the notebook files in this repo that work with the latest master.

**Comment by timothydmorton** [comment]
Yes, many are old/outdated; I haven't (ever?) cleaned the notebooks directory out.  https://github.com/timothydmorton/isochrones/blob/master/notebooks/sun-compare.ipynb should work with current master if you want a demo.  Also, the rtd-hosted docs should be more or less up to date.
